src/platform/user/authentication/utilities.js

```diff
@@ -105,16 +113,16 @@ function redirect(redirectUrl, clickedEvent) {
 }
 
 export function login(policy, version = 'v0', application = null, to = null) {
-  return redirect(
-    loginUrl(policy, version, application, to),
-    'login-link-clicked-modal',
-  );
+  const url = loginUrl(policy, version, application, to);
+  setForceAuth();
+  return redirect(url, 'login-link-clicked-modal');
 }
 
 export function autoLogin() {
   const url = sessionTypeUrl('idme', 'v1', undefined, undefined, {
     inbound: 'true',
   });
+  setForceAuth();
```

Comment:
is this only going to force someone to enter credentials if there's an error state? I.e., I get redirected to autologin, and am only prompted to log in again if something fails?

Reply:
yeah.  The idea is that we'll allow SSOe to short circuit the request and return whatever SAML response they have cached once.  If it works, great the user is logged in.  However, if it fails for any reason, any subsequent attempt to login should have the `?force=true` query parameter set, meaning the user should be forced to reenter their credentials.